src/components/Tooltip/tooltipPropTypes.js

```diff
@@ -42,6 +59,7 @@ const defaultProps = {
     renderTooltipContent: undefined,
     renderTooltipContentKey: [],
     shouldHandleScroll: false,
+    shouldUseMultilinePositioning: false,
```

Comment:
Let's remove this prop. I think we want to have this feature enabled for all tooltips